### PR TITLE
Fix test cases with Eventually semantics

### PR DESCRIPTION
**Pull request by lujiajing1126**
Signed-off-by: Megrez Lu <lujiajing1126@gmail.com>

This PR tends to fix test cases failures due to async write.

Before this PR, integration tests randomly failed.

### CODE REVIEW

File: test/cases/measure/data/data.go [hunk 1]
```diff
@@ -26,16 +26,16 @@ import (
 	"time"
 
 	"github.com/google/go-cmp/cmp"
-	g "github.com/onsi/ginkgo/v2"
-	gm "github.com/onsi/gomega"
+	. "github.com/onsi/ginkgo/v2"
```
Comment (hanahmily) [inline test/cases/measure/data/data.go:29]: ci might fails

Comment (lujiajing1126) [reply test/cases/measure/data/data.go:29]: > ci might fails

Oh, I see the linter issues... Let me revert back this part

Comment (lujiajing1126) [reply test/cases/measure/data/data.go:29]: But It seems we miss format for the test package?

**Review by hanahmily — APPROVED** [review]

File: test/cases/measure/data/data.go (hunk 1 above)
Comment (hanahmily) [reply test/cases/measure/data/data.go]: No. It doesn't ignore it. But we have to make the order deterministic.

**Pull request merged**